examples/wolf_sheep/wolf_sheep/server.py

```diff
@@ -42,7 +42,11 @@ def wolf_sheep_portrayal(agent):
 
 canvas_element = CanvasGrid(wolf_sheep_portrayal, 20, 20, 500, 500)
 chart_element = ChartModule(
-    [{"Label": "Wolves", "Color": "#AA0000"}, {"Label": "Sheep", "Color": "#666666"}]
+    [
+        {"Label": "Wolves", "Color": "#AA0000"},
+        {"Label": "Sheep", "Color": "#666666"},
+        {"Label": "Grass", "Color": "#00AA00"},
```

Comment:
I was wondering how the color here is different from https://github.com/projectmesa/mesa/blob/68b6631abd28c2780ab90a3254e6355a3f0de94d/examples/wolf_sheep/wolf_sheep/server.py#L31. Turns out that, that array is a color gradient. See this commit https://github.com/projectmesa/mesa/commit/da194e3556e7a8e61af05da3f726ef42ec7c7c0c.
I think that line is under-documented, and can be clarified by adding a comment that the array is a color gradient.

That said, I am fine with #00AA00.

Reply:
Raised #1183.